### PR TITLE
New desktop portfolio branch

### PR DESCRIPTION
**Pull request by efoosag**
I could not understand why the desktop-portfolio-branch was not  working, so I created another one

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -477,3 +519,170 @@ button:active {
...
+    margin-right: 12px;
+    margin-bottom: 12px;
+  }
+
+  .card-odd img {
+    float: right;
+    width: 50%;
+    margin-right: 12px;
+    margin-bottom: 12px;
+  }
```
Comment (medaminedev66) [inline styles.css:608]: - [ ] It would be better to add margins between elements to improve your UI design:
image

File: index.html [hunk 2]
```diff
@@ -75,8 +78,8 @@ <h2>Tonic</h2>
           </div>
         </div>
         <div class="card">
-          <div class="card1">
-            <img src="./images/Snapshoot-Portfolio1.svg" alt="Card 1" />
+          <div class="card-even">
+            <img src="./images/Snapshoot-Portfolio1.svg" alt="Card 2" />
           </div>
           <h2>Multi-Post Stories</h2>
           <ul>
```
Comment (medaminedev66) [inline index.html:85]: - Kindly add gaps between skills so your design matches the Figma design:
Your design:
image

File: styles.css [hunk 3]
```diff
@@ -477,3 +519,170 @@ button:active {
...
+    margin: 14px 260px 0 260px;
+  }
+
+  h2 {
+    display: flex;
+  }
+
+  .card {
+    display: inline-block;
+  }
```
Comment (medaminedev66) [inline styles.css:594]: - [ ] Following the project requirements, you should stick to the Figma design as much as you can. In the template that you have chosen the position of the image in the card is different in each card. which in your design is still at the same position in all your cards. See the comparison below 👇:
- Your design:
image


- Figma design:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @efoosag , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️Your desktop version design is amazing and matches the Figma design
- ✔️Git/GitHub flow best practices have been followed

## Required Changes ♻️

- [x] Kindly try to solve the conflict before requesting another review to be able to merge your pull request when it's approved.
_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (efoosag) [reply styles.css]: done

File: index.html (hunk 2 above)
Comment (efoosag) [reply index.html:139]: done

File: styles.css (hunk 3 above)
Comment (efoosag) [reply styles.css:648]: done

**Comment by efoosag** [comment]
Changes Made.
I await your re-review.
Thank you for the review.

File: index.html [hunk 4]
```diff
@@ -189,89 +44,265 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+          <div class="see-project">
+            <a href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-even">
+            <img src="./images/Snapshoot2.svg" alt="Card 2" />
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <ul>
```
Comment (MrBrN197) [inline index.html:85]: - [ ] Hi, you've done well to fix all the other issues from the previous reviewer. Although I'm looking at your project and I don't think this particular issue mentioned by the reviewer was addressed. please fix this small issue and your project should be complete. 
- [ ] look at the little badges with the text "HTML", "CSS", "javascript". you can see that they are bundled together but the are supposed to have some space between them. 

Your Design | Reference Design
-|-
image|image

- [ ] notice that on the Figma there is space between the badges. maybe adding a `gap: ` property on the CSS should fix this

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
Hi @efoosag  👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @efoosag, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**